### PR TITLE
Removing leading newline in code block

### PR DESCRIPTION
**Pull request by spier**
This ain't a biggie but I was trying to fix a little formatting problem in the README.
However not sure if my change has the desired effect?

Here the thing that I am trying to fix:

leading-newline

## Proposed Changes

- minor formatting issue in the README

### Checklist

- [ ] Read the [**CONTRIBUTING**](CONTRIBUTING.md) document.
- [ ] Read the [**CODE OF CONDUCT**](CODE_OF_CONDUCT.md) document.
- [ ] Add tests to cover changes.
- [ ] Ensure your code follows the code style of this project.
- [ ] Ensure CI and all other PR checks are green OR
    - [ ] Code compiles correctly.
    - [ ] Created tests which fail without the change (if possible).
    - [ ] All new and existing tests passed.
- [ ] Add your changes to `Unreleased` section of [CHANGELOG](CHANGELOG.md).
- [ ] Improve and update the [README](README.md) (if necessary).
- [ ] Ensure [documentation](./DOCS.md) is up-to-date. The same file will be updated in [plugin index](https://github.com/drone/drone-plugin-index/blob/master/content/meltwater/drone-cache/index.md) when your PR is accepted, so it will be available for end-users at http://plugins.drone.io.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,8 +219,8 @@ $ docker run --rm \
 ## Development
 
 [embedmd]:# (tmp/make_help.txt)
```
Comment (spier) [inline README.md:221]: @kakkoyun what does this do? I wonder if this has to do with the extra newline that I am seeing. ahead of this code block.

Comment (kakkoyun) [reply README.md:221]: Probably that's the reason. This an annotation for a tiny program (https://github.com/campoy/embedmd) that we run to embed code pieces or generated things into markdown files. It's quite handy. Especially for the dynamic content. You can find the detailed usage in the Makefile. https://github.com/meltwater/drone-cache/blob/3172b74d6b3e9c1905b0a18f7fcb69604483609b/Makefile#L66-L68

Comment (kakkoyun) [reply README.md:221]: I think if you remove leading `\n` in `printf` statement, it should work.

Comment (spier) [reply README.md:221]: I see. It doesn't look like we are generating any dynamic help text here at all, are we?

Comment (spier) [reply README.md:221]: I removed the `\n` now as you suggested. 

However I don't understand how this is working.
I ran `make README.md` however that lead to a duplicate code block in the `## Development` section.

You see me a bit confused :)

**Comment by kakkoyun** [comment]
CI is broken because of an unrelated issue https://github.com/meltwater/drone-cache/pull/119/files#r429900969.
So merging this one.

**Pull request merged**